source/appModules/soffice.py

```diff
@@ -318,6 +319,43 @@ def _get_locationText(self):
 class SymphonyDocument(CompoundDocument):
 	TextInfo = SymphonyDocumentTextInfo
 
+	# override base class implementation because that one assumes
+	# that the text retrieved from the text info for the text unit
+	# is the same as the text that actually gets removed, which at
+	# least isn't true for Writer paragraphs when removing a word
+	# followed by whitespace using Ctrl+Backspace
+	def _backspaceScriptHelper(self, unit: str, gesture: inputCore.InputGesture):
+		try:
+			oldInfo = self.makeTextInfo(textInfos.POSITION_CARET)
+			if not isinstance(oldInfo, SymphonyDocumentTextInfo):

```

Comment:
It seems text info for this class is explicitly set to `SymphonyDocumentTextInfo`, so the `isinstance` call seems unnecessary. Have you encountered a case in which text info was of a different type here?

Reply:
No, I haven't. Looking at it again, that check is indeed unnecessary and I've dropped it now. Thanks!